packages/slice-machine/src/modules/selectedSlice/reducer.ts

```diff
@@ -189,6 +190,16 @@ export const selectedSliceReducer: Reducer<
       const { newSliceName } = action.payload;
       return renamedComponentUI(prevState, newSliceName);
     }
+
+    case getType(updateSelectedSliceMocks): {
+      if (!prevState) return prevState;
+      const { mocks } = action.payload;
+      return {
+        ...prevState,
+        mock: mocks,
+        screenshots: prevState?.screenshots,
```

Comment:
Do we need this? Looks like it can be omitted

Reply:
I think so :)